string NHSNumber
string DoctorID
string FirstName
string Surname
string DoB
string Email
string Address
string Discount
string Prescription_Info

